ns_pipelines/participant_demographics/run.py

```diff
@@ -0,0 +1,88 @@
+""" Extract participant demographics from HTML files. """
```

Comment:
I wonder if given that we now have a class inside `run.py` with a `run()` method, if we should rename `run.py` to something else? `pipeline.py`?